indico/modules/events/timetable/controllers.py

```diff
@@ -109,3 +113,17 @@ def _process_PATCH(self):
         if updates:
             update_timetable_entry(self.timetable_entry, updates)
         return jsonify()
+
+
+class RHTimetable(RHConferenceBaseDisplay):
+    def _checkParams(self, params):
+        RHConferenceBaseDisplay._checkParams(self, params)
+        self.event = self._conf
```

Comment:
I'd rather keep using `self._conf` for this. I know we used this assignment quite a lot when we didn't have an `Event` object but now it's just really confusing